linking_static.go

```diff
@@ -6,5 +6,7 @@
 
 package lxc
 
-// #cgo LDFLAGS: -static -llxc -lseccomp -lutil -lcap
+// #cgo LDFLAGS: -static -llxc -lutil -lselinux -lcap -lseccomp
+// #cgo pkg-config: lxc libseccomp libselinux libcap gnutls
```

Comment:
maybe `#cgo pkg-config: --static`?  Haven't had a chance to try so no idea if it works

Reply:
that causes problems with gnutls, it wants a shared version of libunistring